README.md

```diff
@@ -187,6 +351,62 @@ jobs:
       NEXUS_PASSWORD: ${{ secrets.NEXUS_PASSWORD }}
 ```
 
+### Shared Deploy Workflow
+
+Workflow to deploy Mulesoft projects to CloudHub 1.0. See [.github/workflows/shared_deploy_cloudhub_1_0.yml](.github/workflows/shared_deploy_cloudhub_1_0.yml)
+
+#### Usage
+
+Create a new environment for deployment and set the needed environment variables, secrets.
+
+```yml
+- uses: nimblehq/mulesoft-actions/.github/workflows/shared_deploy_cloudhub_1_0.yml@v1
+  with:
+    # CloudHub application name
+    # Required
+    cloudhub_application_name: my-app
+
+    # CloudHub region
+    # Required
+    cloudhub_region: ap-southeast-1
+
+    # CloudHub environment
+    # Required
+    cloudhub_environment: DEV
```

Comment:
Should we add the `mule_runtime_version` here @hoangmirs

Reply:
Added in https://github.com/nimblehq/mulesoft-actions/pull/2/commits/b04d52c18f265bc94e8afe85144358667dc8587a